src/bms/parse/model/obj.rs

```diff
@@ -0,0 +1,268 @@
+//! Definitions of the note object.
+use crate::bms::{
+    Decimal,
+    command::{
+        ObjId,
+        channel::{Channel, Key, NoteKind, PlayerSide},
+        time::{ObjTime, Track},
+    },
+};
+
+/// An object on the score.
```

Comment:
```suggestion
/// An object playing sound on the score.
```
We might as well improve that this doc to be more specific.